torch/csrc/jit/fusion_compiler.h

```diff
@@ -28,7 +36,7 @@ struct TensorDesc {
   : TensorDesc(type, TensorDesc::findContiguous(sizes, strides)) {}
   TensorDesc(const at::Tensor& t)
     : TensorDesc(t.type().scalarType(), t.sizes(), t.strides()) {}
-  TensorDesc(TensorTypePtr type)
+  TensorDesc(CompleteTensorTypePtr type)
```

Comment:
This constructor is pretty weird. Aren't you always constructing TensorDesc from real tensors, because you want to actually run the fusion kernel?

Reply:
I think it was used previously and is no longer needed. I will remove it.

Reply:
My bad, it is still use. I create some CompleteTensorTypes manually out of `map_size`.